### PR TITLE
feat: Endpoints refreshed with new Request status model and Protobuf models added

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: src/neptune_api/proto/__init__.pyi
Comment (AleksanderWWW) [inline src/neptune_api/proto/__init__.pyi:1]: What's the point of this file? Cannot those imports be in the regular `__init__.py`?

File: src/neptune_api/api/data_ingestion/check_request_status.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     AuthenticatedClient,
     Client,
 )
-from ...models.ingest_response import IngestResponse
+from ...proto.neptune_pb.ingest.v1.request_status_pb2 import RequestStatus
```
Comment (AleksanderWWW) [inline src/neptune_api/api/data_ingestion/check_request_status.py:16]: Is it possible to replace such cases with an explicit import like `package1.package2.etc`?

File: src/neptune_api/proto/__init__.pyi
Comment (Raalsky) [reply src/neptune_api/proto/__init__.pyi:1]: I'm just following what `protoc` and `protol` tools gives me. The second one fixes the relative imports issue.

File: src/neptune_api/api/data_ingestion/check_request_status.py (hunk 1 above)
Comment (Raalsky) [reply src/neptune_api/api/data_ingestion/check_request_status.py:16]: Yes, this one shouldn't be a problem

**Review by AleksanderWWW — APPROVED** [review]

**Pull request merged**